### ISSUE TITLE
Error revealed when trying out test code

### ISSUE THREAD

**Issue by chufz** [kruvelab/MS2Quant#5]
Dear kruvelab,

we tried out the code provided, however, we revealed the following error when using the test code:

```
! no applicable method for 'predict' applied to an object of class "c('train', 'train.formula')"
```

**For the MS2Quant_predict_IE function:**

This error is related to line 1125 in functions.R

**For the MS2Quant_quantify function:**

This error seems to be related to line 925 in functions.R  :

```
# 3) Predict logIE values with MS2Quant for calibrants
  calibrants_structural_FP <- calibrants_structural_FP %>%
    mutate(pred_logIE = predict(MS2Quant, newdata = calibrants_structural_FP))
```

Have you experienced any similar error or can you tell with which version of xgboost the loaded model "model_MS2Quant_xgbTree_allData.RData" was build?

**Comment by helensepman** [comment]
Hi! 
I think the reason is that the _caret_ package is not listed as one of the dependencies. You can try installing _caret_ and reading it in as a library before running MS2Quant. Let me know if it helps.

**Comment by chufz** [comment]
Yes, this solves that error message, can you please add _caret_ to the dependencies? 

Do have the possibility to figure out which version of xgboost was used for the training of the model in "model_MS2Quant_xgbTree_allData.RData"?

**Comment by helensepman** [comment]
It will be added but as the package is currently being updated, it will take some time. 
I have not figured out a way to check the xgboost version yet, but I will get back to you if I find it.

**Comment by chufz** [comment]
Thanks a lot for your time and help!
